firestore/query.go

```diff
@@ -1239,14 +1242,17 @@ type docIterator interface {
 	stop()
 }
 
-// GetExplainMetrics returns query explain metrics.
+// ExplainMetrics returns query explain metrics.
 // This is only present when ExplainOptions run option is provided in the query.
-func (it *DocumentIterator) GetExplainMetrics() (*ExplainMetrics, error) {
+// when [ExplainOptions] is added to the query (see [Query.WithRunOptions]),
```

Comment:
Line 1243 should read "This is only present". The "when" of this line continues that sentence.

Reply:
Done